forms-web-app/src/controllers/eligibility/planning-department.js

```diff
@@ -1,3 +1,16 @@
+const PLANNING_DEPARTMENT = 'eligibility/planning-department';
+
+const LPAS = [
```

Comment:
Happy for this to go in like this, but please create a ticket to get data from the appeals service API